bi_sql_editor/models/bi_sql_view.py

```diff
@@ -324,7 +329,7 @@ def button_set_draft(self):
             sql_view.pivot_view_id.unlink()
             sql_view.search_view_id.unlink()
             if sql_view.cron_id:
-                sql_view.cron_id.unlink()
```

Comment:
As you remove unlink here, you should add if the bi.sql.view is deleted I think.

Apart that point, LGTM. Very ligther solution, and nice improvment ! thanks !

Reply:
sure, thank you for noticing 👍🏿 just applied the changes, we should be good now 😄